web_responsive/views/web.xml

```diff
@@ -206,12 +209,19 @@
 
             <div class="panel-default app-drawer-app-panel" id="appDrawerAppMenu">
                 <div class="panel-heading" id="appDrawerAppPanelHead">
-                    <h4 class="app-drawer-panel-title">
-                        <a href="#" class="app-drawer-icon-close drawer-toggle">
-                            <i class="fa fa-lg fa-chevron-left" />
-                            Apps
+                    <div class="col-xs-6">
+                        <h4 class="app-drawer-panel-title pull-left">
+                            <a href="#" class="app-drawer-icon-close drawer-toggle">
+                                <i class="fa fa-lg fa-chevron-left" />
```

Comment:
You'll probably get dirty translations with this. Better use a div for fontawesome icons or add `t-translation="off"` please.